cmd/fs-v1-helpers.go

```diff
@@ -418,11 +409,6 @@ func fsRenameFile(ctx context.Context, sourcePath, destPath string) error {
 		return err
```

Comment:
We could remove the `checkPathLength` calls in fsRenameFile, we are already doing this in renameAll()

Reply:
This PR is not going to do that @Praveenrajmani - the intention was to avoid Stat() calls in this PR